Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/AnalyticsSettings.kt

```diff
@@ -15,6 +19,21 @@ class AnalyticsSettings @Inject constructor(
         context.getSharedPreferences("analytics_localdata", Context.MODE_PRIVATE)
     }
 
+    val previousExposureRiskMetadata = FlowPreference(
+        preferences = prefs,
+        key = PREVIOUS_EXPOSURE_RISK_METADATA,
+        reader = { key ->
+            getString(key, null)?.let { prefString ->
+                prefString.decodeBase64()?.toByteArray()?.let {
+                    ExposureRiskMetadata.parseFrom(it)
```

Comment:
What happens if there is a change (accidental?) to the protobuf files or the way they are generated. Could this throw an exception? Will we remember that there is data that might throw an exception then?

Would it be okay for logic consuming this value, to receive `null` in that case, so we could just wrap this with a warm & comfy try-catch?

Reply:
logic handling with null is fine here, also added a test that validates changes in the proto definition that we need to handle